src/browser/components/preferences/jar-mn.patch

```diff
@@ -1,10 +1,11 @@
 diff --git a/browser/components/preferences/jar.mn b/browser/components/preferences/jar.mn
-index 118709048e7de13f6ac10d0047e446b72303428f..c8cc2d7ee551b96be668a7844dab1db5abc9d684 100644
+index 118709048e7de13f6ac10d0047e446b72303428f..1bed8626c2d7e1e08885a07aac777bec02f6e4dd 100644
 --- a/browser/components/preferences/jar.mn
 +++ b/browser/components/preferences/jar.mn
-@@ -26,3 +26,5 @@ browser.jar:
+@@ -26,3 +26,6 @@ browser.jar:
     content/browser/preferences/widgets/setting-control.mjs              (widgets/setting-control/setting-control.mjs)
     content/browser/preferences/widgets/setting-group.mjs                (widgets/setting-group/setting-group.mjs)
     content/browser/preferences/widgets/setting-group.css                (widgets/setting-group/setting-group.css)
 +
 +   content/browser/preferences/zen-settings.js
++   content/browser/preferences/zen-air-traffic-control.js
```

Comment:
Could we please have all these settings in the parent window as a dialog? Trying to avoid the preferences page as much as possible.

Reply:
I think a need a little bit more clarity. Where should I put the affordance that would open up the dialog? Do you have an example where you've already done this in Zen?

Reply:
Maybe it's fine, we'll give it a try as it is 👍🏽

Reply:
@meenie unfortunately, I can't think of an example where we are currently doing this on zen. 

But it would ideally just be the same XHTML contents but instead of being in the settings, it's opened via https://searchfox.org/mozilla-central/source/browser/base/content/browser.js#5250.

One example of this modal is when pressing CMD+Q on mac